waku/v2/protocol/rln/group_manager/group_manager.go

```diff
@@ -1 +1,21 @@
 package group_manager
+
+import (
+	"context"
+
+	"github.com/waku-org/go-zerokit-rln/rln"
+)
+
+type GroupManagerI interface {
+	Start(ctx context.Context) error
+	IdentityCredentials() (rln.IdentityCredential, error)
+	MembershipIndex() rln.MembershipIndex
+	Stop() error
+}
+
+type GMDetails struct {
```

Comment:
to avoid stuttering since the package is already named `group_manager`, this can be renamed to `Details`. i.e.:   `group_manager.Details`